src/main/components/Branding.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Created by tsadler on 02/05/2017.
+ */
+import React, { Component } from "react";
+import styled from "styled-components";
+import Card from "./Card";
+
+const Container = styled(Card)`
+  display: flex; 
+  flex-direction: row;
+  margin-left : 30px;
+  margin-right : 45px;
+  margin-bottom: 20px;
+  align-items: center;
+  justify-content: center;
+`;
+
+class Branding extends Component {
+  render() {
+    return (
+      <Container>
+
+        <img
+          style={{
+            width: 250,
+            height: 150
```

Comment:
these images should have an 'alt' attribute. For screen-readers/incase image fails to load